tests/test_interops/test_trackmate_xml.py

```diff
@@ -0,0 +1,583 @@
+import io
+from copy import deepcopy
+
+import networkx as nx
+import networkx.algorithms.isomorphism as iso
+import pytest
+from lxml import etree as ET
+
+import geff.interops.trackmate_xml as tm_xml
+
+
+def is_equal(obt, exp):
```

Comment:
This will be useful for other people and can live in a more central place. I would put it either in utils or testing.utils if that exists.